src/libraries/System.Runtime.InteropServices.RuntimeInformation/tests/CheckPlatformTests.cs

```diff
@@ -116,5 +116,29 @@ public void CheckOSPlatform()
             Assert.Equal(0, defaultObj.GetHashCode());
             Assert.Equal(defaultObj.GetHashCode(), conObj.GetHashCode());
         }
+
+        [Fact, PlatformSpecific(TestPlatforms.FreeBSD)]
```

Comment:
Nit: Could you please move this up so that it is together with the other `  [Fact, PlatformSpecific` tests.